### PR TITLE
Add table of plugins.

### PR DESCRIPTION
**Pull request by denysdovhan**
Add table of used plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,6 @@
 
 > typography set for fixing common issues
 
-This packages handles for you:
-
-* quotes, apostrophes, ellipses, and spaces as well as dashes
-* plus arrows, copyright, trademark, math and currency symbols
-
 Can be used as [textr](https://github.com/shuvalov-anton/textr) plugin too.
```
Comment (iamstarkov) [inline README.md:15]: I would like to keep it

File: README.md [hunk 2]
```diff
@@ -44,10 +39,45 @@ Default: `{}`
 
 Mainly because of `options.locale` importance. You know better which locale you text belongs to.
 
+## Batteries included
+
+This packages handles for you:
+
+| Package           | Description                     | Example              |
+| :---------------- | :------------------------------ | :------------------- |
+| [`typographic-apostrophes`][apostrophes] | Apostrophes in contractions and for possessive case. | `rock'n'roll` → `rock’n’roll` |
```
Comment (iamstarkov) [inline README.md]: maybe use non-breake spaces in example column: `smth` → `something`?

Comment (iamstarkov) [inline README.md]: screen shot 2015-11-28 at 14 48 59

File: README.md (hunk 1 above)
Comment (denysdovhan) [inline README.md:15]: ok, I'll keep it

File: README.md (hunk 2 above)
Comment (denysdovhan) [inline README.md]: right, should be better

File: README.md (hunk 1 above)
Comment (iamstarkov) [inline README.md:15]: thank you

File: README.md (hunk 2 above)
Comment (iamstarkov) [inline README.md]: awsm

**Comment by denysdovhan** [comment]
Done.

**Comment by iamstarkov** [comment]
👍thank you

**Comment by denysdovhan** [comment]
U R welcome

**Pull request merged**